ArduPlane/quadplane.cpp

```diff
@@ -269,7 +269,7 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
     // @Bitmask: 8: Mtrs_Only_Qassist-in tailsitters only uses VTOL motors and not flying surfaces for QASSIST
     // @Bitmask: 10: Disarmed Yaw Tilt-enable motor tilt for yaw when disarmed
     // @Bitmask: 11: Delay Spoolup-delay VTOL spoolup for 2 seconds after arming
-    // @Bitmask: 12: Disable Qassist-based on synthetic airspeed even if airspeed sensor is used
+    // @Bitmask: 12: Disable speed based Qassist when using synthethic airspeed estimates
```

Comment:
Would it be clearer to say "Disable speed based QAssist when not using an airspeed sensor" or even
 "Disable speed based QAssist when there is no airspeed sensor or ARSPD_USE = 0".

Reply:
both suggestions are incorrect....synthetic airspeed may be in use as the cannonical airspeed even with an airspeed sensor....

Reply:
Is this statement wrong: So actually the independent concept of "synthetic airspeed" doesn't really exist. There is cannonical airspeed which is synthesized from all available sources, which will include physical airspeed sensor(s) if they exist or be purely synthetic otherwise.

Reply:
incorrect....synthetic airspeed is a computation done in both nav filters all the time...cannonical airspeed is the airspeed being used by libraries WHEN they want cannonical airspeed....TECS for example does not use the cannonical airspeed if its synthetic based unless the TECS_SYNAIRSPEED param is enabled...it check for that in the code....Q_OPTIONS bit 12 forces Q_ASSIST code to determine if the cannonical airspeed is synthetic or not , in order to decide on whether to assist below Q_ASSIST_SPEED or not....

Reply:
> incorrect....synthetic airspeed is a computation done in both nav filters all the time...cannonical airspeed is the airspeed being used by libraries WHEN they want cannonical airspeed....TECS for example does not use the cannonical airspeed if its synthetic based unless the TECS_SYNAIRSPEED param is enabled...it check for that in the code....Q_OPTIONS bit 12 forces Q_ASSIST code to determine if the cannonical airspeed is synthetic or not , in order to decide on whether to assist below Q_ASSIST_SPEED or not....

Sorry to harp on this @Hwurzburg - so then where would I find this "synthetic airspeed" in the logs?

Reply:
its not logged, many internal variables are not...